.github/workflows/module3.yml

```diff
@@ -0,0 +1,21 @@
+name: C++ with CMake CI
+
+on: [push, pull_request]
+
+jobs:
+  pizzas:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@master
+    - name: mkdir build
+      run: mkdir build
+    - name: Compile
+      working-directory: build
+      run: |
+         cmake ..
+         make
+    - name: Run tests
+      working-directory: build
+      run: ctest -V
```

Comment:
there are no tests registered in CMakeLists.txt. Either register them with add_test and enable_testing or run a binary here by name.